setup.py

```diff
@@ -57,10 +56,7 @@
             "o2r.recon.new=Ot2Rec.main:create_recon_yaml",
             "o2r.recon.run=Ot2Rec.main:run_recon",
 
-            "o2r.recon.new_ext=Ot2Rec.main:create_recon_yaml_stacked",
-            "o2r.recon.run_ext=Ot2Rec.main:run_savurecon",
-
-            "o2r.savu.new=Ot2Rec.main:create_savurecon_yaml",
+            "o2r.savu.new=Ot2Rec.main:create_recon_yaml_stacked",
```

Comment:
should this be `o2r.savu.new=Ot2Rec.main:create_savurecon_yaml`?

Reply:
No - it's correct, but I'd need to change the docstring for sure, it still says for IMOD now (but it does Savu things)